webplugin/js/app/kommunicate.js

```diff
@@ -208,6 +208,13 @@ $applozic.extend(true, Kommunicate, {
         KommunicateUI.showChat();
         KommunicateUI.hideFaq();
     },
+    reloadWidget: function () {
+        ALStorage.clearAppHeaders();
+        window.$applozic.fn.applozic('reInitialize', {
+            userId: kommunicate._globals.userId,
```

Comment:
in case if user have a passed password during initial login, will this work?

Reply:
made required changes for password